tools/create-yorkie-app/.env

```diff
@@ -0,0 +1,2 @@
+VITE_YORKIE_API_ADDR='https://api.yorkie.dev'
+VITE_YORKIE_API_KEY='ckfvbld047aeajg7f01g'
```

Comment:
Since each example project needs yorkie API server and key, I created this static key for create-yorkie-app. Please share if there is a better approach. 😅

Reply:
I don't think we need to provide an api key by default, and making an api key public is not good.

So I think it's okay to let users use their api keys.

If we have to, it's enough to print a message to get an api key after creating the project or specify in the README for each example about api key.